### ISSUE TITLE
musllinux: giac build fails with "undefined reference to `libintl_gettext'"

### ISSUE THREAD

**Issue by mkoeppe** [passagemath/passagemath#225]
```
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /usr/lib/gcc/i586-alpine-linux-musl/13.2.1/../../../../i586-alpine-linux-musl/bin/ld: icas.o: in function `print_help_text(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&)':
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /host/sage-musllinux_1_2_i686/var/tmp/sage/build/giac-1.9.0.93+passagemath-2024-10-29/src/src/icas.cc:406:(.text+0x1f2c): undefined reference to `libintl_gettext'
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /usr/lib/gcc/i586-alpine-linux-musl/13.2.1/../../../../i586-alpine-linux-musl/bin/ld: /host/sage-musllinux_1_2_i686/var/tmp/sage/build/giac-1.9.0.93+passagemath-2024-10-29/src/src/icas.cc:414:(.text+0x213c): undefined reference to `libintl_gettext'
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /usr/lib/gcc/i586-alpine-linux-musl/13.2.1/../../../../i586-alpine-linux-musl/bin/ld: /host/sage-musllinux_1_2_i686/var/tmp/sage/build/giac-1.9.0.93+passagemath-2024-10-29/src/src/icas.cc:423:(.text+0x2414): undefined reference to `libintl_gettext'
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /usr/lib/gcc/i586-alpine-linux-musl/13.2.1/../../../../i586-alpine-linux-musl/bin/ld: /host/sage-musllinux_1_2_i686/var/tmp/sage/build/giac-1.9.0.93+passagemath-2024-10-29/src/src/icas.cc:396:(.text+0x2885): undefined reference to `libintl_gettext'
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /usr/lib/gcc/i586-alpine-linux-musl/13.2.1/../../../../i586-alpine-linux-musl/bin/ld: icas.o: in function `is_help_text(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >&)':
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /host/sage-musllinux_1_2_i686/var/tmp/sage/build/giac-1.9.0.93+passagemath-2024-10-29/src/src/icas.cc:268:(.text+0x6c89): undefined reference to `libintl_gettext'
    [giac-1.9.0.93+passagemath-2024-10-29]   [spkg-install] /usr/lib/gcc/i586-alpine-linux-musl/13.2.1/../../../../i586-alpine-linux-musl/bin/ld: icas.o:/host/sage-musllinux_1_2_i686/var/tmp/sage/build/giac-1.9.0.93+passagemath-2024-10-29/src/src/icas.cc:268: more undefined references to `libintl_gettext' follow
```
https://github.com/passagemath/passagemath/actions/runs/11586190416/job/32257122163#step:9:6500

**Comment by mkoeppe** [comment]
This may have been introduced with the debian removal of `intl/`

**Comment by mkoeppe** [comment]
Fixed by #221
